sdk/cmake/CMakeMacros.cmake

```diff
@@ -162,14 +162,19 @@ function(add_link)
     set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/${_LINK_NAME}.lnk PROPERTIES GENERATED TRUE)
 endfunction()
 
+#
+# WARNING!
+# Please keep the numbering in this list in sync with
+# boot/bootdata/packages/reactos.dff.in
+#
```

Comment:
I wonder if there would be a way to parse the path list from the file, instead of hardcoding it... just a thought.

Reply:
Good thought; the current problem indeed being that duplication of information here and in the txtsetup.sif. Having some kind of automatic generation there would be great, but it's beyond the scope of this work.